app/controllers/admin/republishing_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Admin::RepublishingController < Admin::BaseController
   include Admin::RepublishingHelper
+  include ReshuffleMode
```

Comment:
Using this model concern in a controller feels a tad weird. On the other hand it's not really a model concern arguably, we just don't have anywhere more logical to put it 🤔

Reply:
Just realised I missed this first time around, sorry for commenting at second time of asking!

Reply:
Yeah it felt a bit odd. I think we could probably find a way of removing it from the RepublishingController, as it's only used to make `reshuffle_in_progress?` available (could probably extract that to a helper class we can `require_relative` or something).

But the other controllers make use of the `republish_ministers_index_page_to_publishing_api` callbacks which I don't think we can move to a helper class whilst still allowing neatly referencing it in an `after_save`. And rather than spread the code across both a concern and a helper, I like that we've encapsulated this mess in one place 😅 So think this is good to go for now. Thanks!